annotations.Rmd

```diff
@@ -228,28 +249,113 @@ ggplot(economics, aes(date, unemploy)) +
   )
 ```
 
-It's easier to use the `annotate()` helper function which creates the data frame for you: \indexf{annotate}
+This code works, and generates the desired plot, but it is very cumbersome. It would be annoying to have to do this every time you want to add a single annotation, so ggplot2 includes the `annotate()` helper function which creates the data frame for you: \indexf{annotate}
 
-```{r}
+```{r annotate_easyway}
 ggplot(economics, aes(date, unemploy)) + 
   geom_line() + 
   annotate("text", x = xrng[1], y = yrng[2], label = caption,
     hjust = 0, vjust = 1, size = 4
   )
 ```
 
-Annotations, particularly reference lines, are also useful when comparing groups across facets. In the following plot, it's much easier to see the subtle differences if we add a reference line.
+The convenience of the `annotate()` function comes in handy in other situations. For example, a common form of annotation is to highlight a subset of points by drawing larger points in a different colour underneath the main data set. To highlight vehicles manufactured by Subaru you could use this to create the basic plot:
+
+```{r underplot-noshow}
+p <- ggplot(mpg, aes(displ, hwy)) +
+  geom_point(data = filter(mpg, manufacturer == "subaru"), 
+             colour = "orange", size = 3) +
+  geom_point() 
+```
+
+The problem with this is that the highlighted category would not be labelled. This is easily rectified using `annotate()`
+
+`r columns(1, 2/3)`
+```{r underplot}
+p + annotate(geom = "point", x = 5.5, y = 40, 
```

Comment:
I think these should fit in a single line? 

If they don't, can you please use the tidyverse long lines style? http://style.tidyverse.org/syntax.html#long-lines

Reply:
Fixed in the current iteration, along with several others I noticed

Reply:
Btw when you resolve a query, you can click the "resolve conversation" button and I'll trust you 😄